futaba/cogs/settings/settings.py

```diff
@@ -1,5 +1,5 @@
 #
-# cogs/settings.py
+# cogs/settings/settings.py
```

Comment:
We should have standard naming for cogs, if those files are needed. This should be `commands.py` or something imo.

Reply:
I agree, having the commands in commands make sense. However I think there should be a core.py that holds the class for the cog and any other file can be named accordingly

Reply:
Sounds good!